src/common/utils/environment-variables.injectable.ts

```diff
@@ -3,14 +3,15 @@
  * Licensed under MIT License. See LICENSE in root directory for more information.
  */
 import { getInjectable } from "@ogre-tools/injectable";
+import nodeEnvInjectionToken from "../vars/node-env-injection-token";
 
 const environmentVariablesInjectable = getInjectable({
   id: "environment-variables",
 
-  instantiate: () => {
+  instantiate: (di) => {
     // IMPORTANT: The syntax needs to be exactly this in order to make environment variable values
     // hard-coded at compile-time by Webpack.
-    const NODE_ENV = process.env.NODE_ENV;
+    const NODE_ENV = di.inject(nodeEnvInjectionToken);
```

Comment:
I think this should be removed from here and used directly from `di.inject(nodeEnvInjectionToken)`. Since the source of `nodeEnv` is no longer environment variable, which is against of what this injectable does.